src/dialects/Dialect.ts

```diff
@@ -37,7 +37,7 @@ export interface IColumnMetadata {
     autoIncrement: boolean;
     indices?: IIndexMetadata[],
     comment?: string;
-    // default?: ;
+    defaultValue?: string;
```

Comment:
I would change `defaultValue` type to `any` since in other dialects the driver could return a different type for the default.
```ts
defaultValue?: any;
```